src/api/seeders/categories.js

```diff
@@ -0,0 +1,31 @@
+const moment = require('moment');
+
+const createdAt = moment().format();
+const updatedAt = createdAt;
+
+module.exports = {
```

Comment:
@murediane These categories you've seeded are too few. Please ask the PO for a list of categories. You can also use the categories found on Medium.